examples/interactivity/popups-format.html

```diff
@@ -106,7 +106,7 @@ <h1 class="as-title">Pop-ups on hover with format</h1>
             format: value => value.toString().replace(/000$/, 'K habitants')
           }
         ]);
-        await populationLayer.addTo(deckMap);
+        populationLayer.addTo(deckMap);
```

Comment:
removing what's not required will be great for the final user, thx for taking the time and effort!